client/src/components/auth/authComponents/RegistrationForm.jsx

```diff
@@ -62,9 +56,11 @@ const RegistrationForm = () => {
     password: yup
       .string()
       .required('Please enter your password')
+      .min(8, 'Password must contain at least 8 characters')
+      .max(20, 'Password is too long')
       .matches(
-        /^(?=.*[A-Za-z])(?=.*\d)(?=.*[@$!%*#?&])[A-Za-z\d@$!%*#?&]{8,}$/,
-        'Password is not strong enough',
+        /^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,20}$/,
+        'Password must contain at least 8 characters, 1 alphabet and 1 number',
```

Comment:
Can be reworded better. Maybe "Password must be least 8 characters and containing at least 1 number."

Reply:
sure